lib/WebService/Stripe.pm

```diff
@@ -54,6 +54,14 @@ method create_customer(Maybe[HashRef] $data, :$headers) {
     return $self->post( "/v1/customers", $data, headers => $headers );
 }
 
+method create_recipient(Maybe[HashRef] $data, :$headers) {
```

Comment:
Would prefer to just make `$data` required. That makes the interface easier to change later if we need to. (Since we don't have to worry about maintaining the optional first argument.)